Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -164,6 +165,7 @@ def bump_formula_pr
     old_formula_version = formula_version(formula, requested_spec)
     old_version = old_formula_version.to_s
     forced_version = new_version.present?
+    new_url ||= update_pypi_url(old_url, new_version) if old_url.match?(%r{^https?://files.pythonhosted.org/})
```

Comment:
I would put this line right before this one: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb#L190

We still need to check if the formula uses either a URL/SHA-256 pair or tag/revision

Reply:
Good catch. Thanks!